### DISCUSSION TITLE
PostgreSQL connection string

### DISCUSSION THREAD

**Discussion by mond23** [jvde-github/AIS-catcher#248] [Q&A] [answered]
Hello!
Could you give an example of what a connection string looks like to connect to a database using the -D argument?
Thank you very much!

**Comment by jvde-github** [comment]
Hi, something like

 -D postgresql://postgres:secret@localhost/ais

But you can leave parts out if not relevant.
For difficult password you might have to url encode the string. I have seen someone bringing this up in another issue.

does this work?

**Reply by mond23** [reply]
Hm, im trying to run with your binaries for Windows. Perhaps something else is needed here. Because i get "DBMS: no support for PostgeSQL build in." in stdout (from DBMS/PostgreSQL.cpp).
Ofc server has been configured and Database has been created

**Comment by jvde-github** [comment, marked as answer]
There was an issue indeed with the binary that was flagged on discord and I fixed. Apologies! Can you download the Edge from the binaries section and try again?

**Reply by mond23** [reply]
Now it's fine.  I can't thank you enough for your cooperation on this!

**Comment by jvde-github** [comment]
The auto build process broke down and support was not included. I might make a new version as you’re the second person wasting time on this problem, which should not be.

thanks for using the program and consider sharing with the community feed :)
